### DISCUSSION TITLE
config to used in Subfolder

### DISCUSSION THREAD

**Discussion by madvik** [prasathmani/tinyfilemanager#1011] [Q&A]
Hi,

Here my configuration
$use_auth = true;
$root_path = $_SERVER['DOCUMENT_ROOT'].'/sauv';
$root_url = 'https://mywebsite.net/sauv';
$http_host = $_SERVER['HTTP_HOST'].'/sauv';

But when I used it, I log in : 
![image](https://user-images.githubusercontent.com/3822529/230567134-92dabc58-f3a6-456d-a71b-04cdb3d6332c.png)

How can we configure it for use it in subfolder ?

thanks

**Comment by prasathmani** [comment]
```php
$root_path = $_SERVER['DOCUMENT_ROOT'].'/sauv';
$root_url = 'sauv';

```
